src/xrGame/DiscordRPC.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+class DiscordRPC
+{
+private:
+    int64_t     start_time          {};
+    string64    build_name          {};
+    string64    current_level_name  {};
+    pcstr       current_task_name   {};
```

Comment:
I don't see any purpose for `current_level_name` to be a member variable.
```suggestion
    s64 start_time{};
    string64 build_name{};
    pcstr current_task_name{};
```